processing.go

```diff
@@ -94,7 +94,7 @@ func (target *ScanTarget) OpenUDP(flags *BaseFlags, udp *UDPFlags) (net.Conn, er
 	if err != nil {
 		return nil, err
 	}
-	return NewTimeoutConnection(nil, conn, flags.Timeout, 0, 0, flags.BytesReadLimit), nil
+	return NewTimeoutConnection(nil, conn, flags.Timeout, flags.RwTimeout, flags.RwTimeout, flags.BytesReadLimit), nil
```

Comment:
Any reason to add these two timeout values?

Reply:
we need to do several consequent scans per ip
by default one read/write awaits result indefinitely
so we need to set read/write timeout to value several times lower than Timeout(it defines how much time thread will spend per ip)